lib/Doctrine/ORM/Query/AST/IndexBy.php

```diff
@@ -33,14 +33,14 @@ class IndexBy extends Node
     /**
      * @var PathExpression
      */
-    public $simpleStateFieldPathExpression = null;
+    public $singleValuedPathExpression = null;
```

Comment:
Since this class isn't marked as `@internal` renaming the property is a BC-break